cypress/e2e/bilans-financiers.cy.js

```diff
@@ -4,23 +4,23 @@ describe('Bilans financiers', () => {
     cy.contains('Données financières');
   });
 
-  it('Should hide bilans when partially confidential', () => {
-    cy.visit('/donnees-financieres/487444697');
-    cy.contains(
-      'Les bilans de cette structure sont accompagnés d’une déclaration de confidentialité.'
-    );
-  });
+  // it('Should hide bilans when partially confidential', () => {
+  //   cy.visit('/donnees-financieres/487444697');
+  //   cy.contains(
+  //     'Les bilans de cette structure sont accompagnés d’une déclaration de confidentialité.'
+  //   );
+  // });
 
-  it('Should display indicateurs financiers', () => {
-    cy.visit('/donnees-financieres/552032534');
-    cy.contains('Date de clôture');
-    cy.contains('31/12/2019');
+  // it('Should display indicateurs financiers', () => {
+  //   cy.visit('/donnees-financieres/552032534');
+  //   cy.contains('Date de clôture');
+  //   cy.contains('31/12/2019');
 
-    cy.contains('Résultat net');
-    cy.contains('2 Mds €');
+  //   cy.contains('Résultat net');
+  //   cy.contains('2 Mds €');
 
-    cy.contains('Source : MEF');
-  });
+  //   cy.contains('Source : MEF');
+  // });
```

Comment:
Either delete or uncomment ?

Reply:
When I visit either :
https://annuaire-entreprises.data.gouv.fr/donnees-financieres/552032534'
or
https://annuaire-entreprises.data.gouv.fr/donnees-financieres/487444697

<img width="1433" alt="image" src="https://github.com/user-attachments/assets/0905c6db-4925-47a8-8db3-90792f180750" />

There is nothing.
The test was working before removing stub because of the data inside the stub.

I commented it until I rework with the new data sources.
What do you think about that ?

Reply:
Source is currently down. Which is annoying as you cannot create mock ^^